src/focus.service.ts

```diff
@@ -509,6 +509,11 @@ export class FocusService {
   private isFocusable(el: HTMLElement) {
     const role = el.getAttribute('role');
     const tabIndex = el.tabIndex;
+    const visible = window.getComputedStyle(el).display !== 'none';
```

Comment:
That works, but do note that we're calling getComputedStyle on every parent now (until we find one with opacity 0, which is not the common case)